### PR TITLE
Fat 993

### PR DESCRIPTION
**Pull request by viktor-draban**
Link to [ticket](https://issues.folio.org/browse/FAT-993)

Added test "When patron and item id's entered at checkout, post a new loan using the circulation rule matched"

### CODE REVIEW

File: mod-circulation/src/main/resources/domain/mod-circulation/features/loans.feature [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+
+    # checkOut
+    * def checkOutResponse = call read('classpath:domain/mod-circulation/features/util/initData.feature@PostCheckOut') { varCheckOutUserBarcode: 55555, varCheckOutItemBarcode: 666666 }
+
+    # get loan and verify
+    Given path 'circulation', 'loans'
+    And param query = '(userId==' + userId + ' and ' + 'itemId==' + itemId + ')'
+    When method GET
+    Then status 200
+    And match checkOutResponse.response.id == response.loans[0].id
```
Comment (alexanderkurash) [inline mod-circulation/src/main/resources/domain/mod-circulation/features/loans.feature:35]: Check policies

Comment (viktor-draban) [reply mod-circulation/src/main/resources/domain/mod-circulation/features/loans.feature]: done

File: mod-circulation/src/main/resources/domain/mod-circulation/features/loans.feature [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+    * call read('classpath:domain/mod-circulation/features/util/initData.feature@PostOverduePolicy') { varOverdueFinePoliciesId: #(overdueFinePoliciesId) }
+    * call read('classpath:domain/mod-circulation/features/util/initData.feature@PostPatronPolicy') { varPatronPolicyId8: #(patronPolicyId) }
+    * call read('classpath:domain/mod-circulation/features/util/initData.feature@PostRequestPolicy') { varRequestPolicyId: #(requestPolicyId) }
+    * call read('classpath:domain/mod-circulation/features/util/initData.feature@PostRulesWithMaterial') { varLoanPolicyId: #(loanPolicyId), varLostItemFeePolicyId: #(lostItemFeePolicyId), varOverdueFinePoliciesId: #(overdueFinePoliciesId), varPatronPolicyId: #(patronPolicyId), varRequestPolicyId: #(requestPolicyId), varMaterialTypeId: #(materialTypeId), varLoanPolicyMaterialId: #(loanPolicyMaterialId), varOverdueFinePoliciesMaterialId: #(overdueFinePoliciesId), varLostItemFeePolicyMaterialId: #(lostItemFeePolicyId), varRequestPolicyMaterialId: #(requestPolicyId), varPatronPolicyMaterialId: #(patronPolicyId) }
```
Comment (alexanderkurash) [inline mod-circulation/src/main/resources/domain/mod-circulation/features/loans.feature:36]: withMaterialType?

File: mod-circulation/src/main/resources/domain/mod-circulation/features/util/initData.feature [hunk 3]
```diff
@@ -0,0 +1,241 @@
...
+    And request patronNoticePolicyEntityRequest
+    When method POST
+    Then status 201
+
+  @PostRequestPolicy
+  Scenario: create request policy
+    * def requestLocalId = call uuid1
+
+    * def policyEntityRequest = read('samples/policies/request-policy-entity-request.json')
+    * policyEntityRequest.id = karate.get('varRequestPolicyId', requestLocalId)
```
Comment (alexanderkurash) [inline mod-circulation/src/main/resources/domain/mod-circulation/features/util/initData.feature:154]: Naming: extRequestPolicyId/intRequestPolicyId

**Pull request merged**